index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>CSS-editor</title>
+    <style>
+        #javascript-input{
+            background-color: black;
+            color: white;
+        }
+    </style>
+<style id="css-style"></style>
+</head>
+<body>
+
+<p>add css here: </p>
+
+<textarea name="css-edit" id="javascript-input" cols="60" rows="30"></textarea>
+
+<h1>Hello world</h1>
+
+<script>
+
+    const javascriptInput = document.querySelector('#javascript-input');
+    const cssStyle = document.querySelector('#css-style');
+
+    javascriptInput.addEventListener('change', e => {
```

Comment:
'keyup' instead of 'change' would be a good option so you don't have to change the focus before it updates